docs/reference-and-specs/command-line-interface.md

```diff
@@ -11,11 +11,13 @@ The `substreams` command line interface (CLI) is the primary user interface and
 The `substreams` CLI exposes many commands to developers enabling a range of features.&#x20;
 
 {% hint style="info" %}
-**Note**: When a package is specified you can use:&#x20;
+**Note**: When a package is specified, it is optional. If you do use it, you can use:&#x20;
 
 * Local `substreams.yaml` configuration files
 * Local `.spkg` package files
 * Remote `.spkg` package URLs
+* Local directory containing a `substreams.yaml` file<br />
```

Comment:
The `<br />` is not needed I'm pretty sure.

Reply:
thats what I thought too but in the ide preview it did, not sure. I'll get rid of it